pkg/controllers/networkinfo/networkinfo_utils.go

```diff
@@ -42,21 +43,31 @@ func deleteSuccess(r *NetworkInfoReconciler, _ *context.Context, o *v1alpha1.Net
 	metrics.CounterInc(r.Service.NSXConfig, metrics.ControllerDeleteSuccessTotal, common.MetricResTypeNetworkInfo)
 }
 
-func setNetworkInfoVPCStatus(ctx *context.Context, networkInfo *v1alpha1.NetworkInfo, client client.Client, createdVPC *v1alpha1.VPCState) {
+func setNetworkInfoVPCStatus(ctx *context.Context, networkInfo *v1alpha1.NetworkInfo, client client.Client, createdVPC *v1alpha1.VPCState) bool {
```

Comment:
Why it returns a bool value, it seems that nobody consumes the returned value?

Reply:
I planned to use the "bool" value to mark if the CR is actually updated or not, and then the outer-caller can use this result to decide if  the metric needs a update accordingly. Unfortunately, I removed my consumer out of this change, and I didn't update the existing callers (`deleteFail, updateFail, updateSuccess`). Maybe I can revert the change?